### DISCUSSION TITLE
Welcome to instagramy Discussions!

### DISCUSSION THREAD

**Discussion by yogeshwaran01** [yogeshwaran01/instagramy#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by prays** [comment]
Hi, I just want to ask.

Is Instagramy package can get number of views of the post given the post code?

**Reply by yogeshwaran01** [reply]
No, there is no property for number of views but it may found in `InstagramPost.post_data` which is python `dict` only for video post.

**Reply by prays** [reply]
Hi,
I just want to clarify.
Is it because instagram only provide views for video post or Instagramy does not provide it yet?

**Comment by Clovra** [comment]
hi
i dont know why but the scripte with the cookies of my session don't match with my reeal number of follower.
do you know why?

this is the scripte :
from instagramy import InstagramUser
  
# Connecting the profile

session_id = "14411687316%3Aws1A5eMHDJGl0u%3A10"
user = InstagramUser('gogle', sessionid=session_id)
 
# printing the basic details like : followers, following
print(user.number_of_followers)
print(user.number_of_followings)
print(user.biography)

**Comment by gouigoui34** [comment]
Hi,
I'm very beginner on python, I started to learn it 3 weeks ago, I try to get the number of followers in an account, it works for "google" (like in the exemple), but not when I try other name account.

This one works : 

>>> from instagramy import InstagramUser

>>> user = InstagramUser('google', from_cache=True)

>>>user.is_verified
>>>True
>>>print ('Nom du compte : ' + str(user))
>>>print('Nombre dabonnés : '+str(user.number_of_followers))
>>> print('Nombre dabonnements : '+str(user.number_of_followings))


But when I replace 'google' with another nameaccount, like 'tomhardy' (random example), it doesn't work anymore and pycharm write me this : 

> Traceback (most recent call last):
>   File "C:\Users\t.mallet\PycharmProjects\Web Scraping Project\venv\web scraper.py", line 4, in <module>
>     user = InstagramUser('tomhardy', from_cache=True)

>   File "C:\Users\t.mallet\PycharmProjects\Web Scraping Project\venv\lib\site-packages\instagramy\InstagramUser.py", line 57, in __init__
>     username, data["entry_data"]["ProfilePage"][0]["graphql"]["user"]

> KeyError: 'ProfilePage'
> 
> Process finished with exit code 1

Should I have to modify something in the instagramy files or something ? Sorry for my  english who's not very well, and I hope someone would be able to help me, thank you very much and I wish you a great day !
